src/main/java/org/opensearch/knn/index/codec/KNN80Codec/KNN80DocValuesConsumer.java

```diff
@@ -210,7 +257,12 @@ public void merge(MergeState mergeState) {
             for (FieldInfo fieldInfo : mergeState.mergeFieldInfos) {
                 DocValuesType type = fieldInfo.getDocValuesType();
                 if (type == DocValuesType.BINARY && fieldInfo.attributes().containsKey(KNNVectorFieldMapper.KNN_FIELD)) {
-                    addKNNBinaryField(fieldInfo, new KNN80DocValuesReader(mergeState));
+                    StopWatch stopWatch = new StopWatch();
```

Comment:
there is some repetitive code. Pls create a function.

Reply:
I personally don't think we should abstract this out, because the StopWatches are only created and used in two locations and each use case would require its own parameter in the method.  In my mind, that over-complicates the code when the use cases are very limited.  Thoughts?